### PR TITLE
Feature/breakpoint redux

### PR DESCRIPTION
**Pull request by eouin**
Add breakpoint utils to add redux state variable for handling watch.media query

### CODE REVIEW

File: utils/breakpoints.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import { setActiveBreakpoint } from '../src/state/app.js';
+
+export const defaultBreakpoints = {
```
Comment (ckoutsiaris) [inline utils/breakpoints.js]: in theme.js we have 5 sizes?

Comment (ianquigley-sap) [reply utils/breakpoints.js]: Yes, we added xxlarge to support the LHS nav.

Comment (eouin) [reply utils/breakpoints.js]: Yep I've seen that ...but why ?

File: utils/breakpoints.js [hunk 2]
```diff
@@ -11,7 +12,7 @@ function dispatchActiveQuery(dispatch, mediaQueryState, action) {
     curr.matches ? curr : prev && prev.matches ? prev : null
   );
 
-  const breakpointName = activeQuery ? activeQuery.name : 'xlarge';
+  const breakpointName = activeQuery ? activeQuery.name : 'xxlarge';
```
Comment (ianquigley-sap) [inline utils/breakpoints.js]: Does this work?  We have xxlarge, but only xlarge defined in the breakpoints list.  Also, we have 4 breakpoints as opposed to 5 originally?

Comment (eouin) [reply utils/breakpoints.js]: yep xxlarge will be the default one.

**Pull request merged**